src/components/navigation/sidebar/sidebar.js

```diff
@@ -0,0 +1,61 @@
+import "./sidebar.css";
+import { Link } from "react-router-dom";
+import { useData } from "../../../context/data-context";
+
+export function Sidebar({ headerToggle }) {
+  const { data } = useData();
+  const { selectedCategory } = data;
```

Comment:
these can be merged in single line for destructuring as following:

const {data: {selectedCategory}} = useData()